packages/react/tests/e2e-storybook/cypress/integration/LocaleModal/LocaleModal.e2e.js

```diff
@@ -20,27 +20,75 @@ describe('LocaleModal | default', () => {
     cy.viewport(1280, 780);
   });
 
+  it('should load with four regions', () => {
+    cy.get('[data-region]').should('have.length', 4);
+
+    cy.get('[data-region="am"]').should('be.visible');
+    cy.get('[data-region="ap"]').should('be.visible');
+    cy.get('[data-region="eu"]').should('be.visible');
+    cy.get('[data-region="mea"]').should('be.visible');
+
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    cy.percySnapshot('LocaleModale | all four regions loaded', {
+      widths: [1280],
+    });
+  });
+
   it('should load the Americas region', () => {
     cy.get('[data-region="am"]').click();
+    cy.get('.bx--locale-modal__locales').should('have.length', 35);
 
     cy.screenshot();
 
     // Take a snapshot for visual diffing
-    cy.percySnapshot('LocaleModal | region selected', {
+    cy.percySnapshot('Footer | Region selected', {
```

Comment:
should this be part of Footer e2e tests then? or should we change the snapshot name back to "LocaleModal"

Reply:
Yeah the footer is also testing some of the LocaleModal functions, but I changed to what it should be here 😅